### ISSUE TITLE
Links opened within Papercut open an Internet Explorer window despite Chrome being my default browser

### ISSUE THREAD

**Issue by capavanni** [ChangemakerStudios/Papercut#106]
I was testing sending some emails with a one-time use link, however the site doesn't yet support Internet Explorer fully so I'd rather open it in Chrome or Firefox.

As I can't right click and copy link from inside the preview, I have to go into the Body section and find the link within the HTML.

I couldn't find anything in settings regarding what browser is used to open links.

Cracking piece of software, thanks for the collective hard work.

**Comment by Jaben** [comment]
Are you saying the "View Html" isn't opening in Chrome or are you saying you want the email preview to be in Chrome?

**Comment by capavanni** [comment]
Hey Jaben, I'm referring to links being rendered within the email body.

So if the email has a link saying "Click here to activate account" I can click that link and have it open on my default browser, rather than Internet Explorer. I've noticed that I can get around this by clicking and dragging the link to a browser of my choice, though.

**Comment by Jaben** [comment]
I cannot reproduce this issue -- which OS are you running?

**Comment by capavanni** [comment]
Hi Jaben, I think I managed to figure it out.

It seems to be setting a `target` of `_blank` on the `a` element that causes the issue.

I double checked that `_blank` is still valid according to specs to make sure I wasn't invoking undefined behaviour and [it seems like it's still supported](https://www.w3.org/TR/html5/browsers.html#valid-browsing-context-name-or-keyword).

I was able to both open with Chrome and IE using the following HTML in the email body.

`
<!DOCTYPE html>
<html>
<head>
	<title>title</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<body>
	<p><a href="https://www.google.com/">Uses Chrome (default browser)</a></p>
	<p><a target="_blank" href="https://www.google.com/">Uses IE</a></p>
</body>
</html>
`

It's not a big issue for me as dragging the link across to Chrome works but I thought it best to report it anyhow :)

Thanks!

(I'm using the latest version of Windows 10)
